### PR TITLE
Migrate block command - Closes #563

### LINKED ISSUE #563: Migrate command block
### Description

Migrate command block
- get

### 
Child of #524

### PR DESCRIPTION
**Pull request by shuse2**
### Description
Migration from
```
lisk get block
```

`npm t`
`./bin/run block:get`

### Review checklist

* It should be merged after #567
* The PR solves #563 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: test/commands/block/get.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+							blockId: block,
+						},
+						placeholder: {
+							blockId: block,
+							message: 'Block not found.',
+						},
+					},
+				]);
+				return expect(printMethodStub).to.be.calledWithExactly(queryResult);
+			});
```
[the remark this answers is not in the document]
Comment (shuse2) [reply test/commands/block/get.test.js]: `.it` is the end of chain, so it cannot have multiple `it` in one test ='(

[the remark this answers is not in the document]
Comment (willclarktech) [reply test/commands/block/get.test.js]: :'(

> Everything here is chainable.

https://github.com/jdxcode/fancy-test#chaining

[the remark this answers is not in the document]
Comment (shuse2) [reply test/commands/block/get.test.js]: 😿

File: src/commands/block/get.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ GetCommand.args = [
 		name: 'blockIds',
 		required: true,
 		description: 'Comma-separated block ID(s) to get information about.',
-		parse: input => input.split(','),
+		parse: input => input.split(',').filter(val => val),
```
Comment (willclarktech) [inline src/commands/block/get.js]: I like `.filter(Boolean)` for this kind of check.

File: test/commands/block/get.test.js [hunk 3]
```diff
@@ -115,5 +116,33 @@ describe('block:get', () => {
 				]);
 				return expect(printMethodStub).to.be.calledWithExactly(queryResult);
 			});
+
+		setupTest()
```
Comment (willclarktech) [inline test/commands/block/get.test.js]: Should we add a similar filter + test to accounts?

Comment (shuse2) [reply test/commands/block/get.test.js]: yes! i will make it on another issue/PR

File: src/commands/block/get.js (hunk 2 above)
Comment (shuse2) [reply src/commands/block/get.js]: something like
`.filter(isString)`?

**Pull request merged**